fe1-web/src/features/social/components/Profile.tsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useMemo } from 'react';
+import { ListRenderItemInfo, StyleSheet, Text, View, ViewStyle } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { useSelector } from 'react-redux';
+
+import { ProfileIcon } from 'core/components';
+import { PublicKey } from 'core/objects';
+import { Border, Color, List, Spacing, Typography } from 'core/styles';
+
+import { SocialHooks } from '../hooks';
+import { Chirp } from '../objects';
```

Comment:
This component reduces code duplication. Previously this code was duplicated: once in the SocialProfile and once in the SocialUserProfile